### PR TITLE
Add new shell commands (sys_time and imu)

### PR DESCRIPTION
**Pull request by gautierhattenberger**
Add 2 new shell commands for sys_time and imu data

### CODE REVIEW

File: sw/airborne/mcu_periph/sys_time.c [hunk 1]
```diff
@@ -89,6 +89,33 @@ void sys_time_update_timer(tid_t id, float duration)
...
+  }
+
+  chprintf(sh, "Current sys_time\r\n");
+  chprintf(sh, "  nb_sec: %lu\r\n", sys_time.nb_sec);
+  chprintf(sh, "  nb_sec_rem: %lu\r\n", sys_time.nb_sec_rem);
+  chprintf(sh, "  nb_tick: %lu\r\n", sys_time.nb_tick);
+  chprintf(sh, "Registered timers\r\n");
+  for (tid_t i = 0; i < SYS_TIME_NB_TIMER; i++) {
+    if (sys_time.timer[i].in_use) {
+      chprintf(sh, "id: %d, duration: %lu, end_time: %lu, cb: 0x%lx\r\n", i,
```
Comment (fvantienen) [inline sw/airborne/mcu_periph/sys_time.c:112]: Would be nice to add names to the timers to identify them easier

**Review by fvantienen — APPROVED** [review]
Cool, should start to use the commandline

**Comment by fvantienen** [comment]
I was thinking it might also be really useful to add the shell as an option on top of the telemetry, such that it doesn't need a separate serial port. That way we could run the shell in the chatbox of the new GCS.

File: sw/airborne/modules/imu/imu.c [hunk 2]
```diff
@@ -334,6 +334,91 @@ static void send_mag_current(struct transport_tx *trans, struct link_device *dev
...
+  chprintf(sh, "    %ld, %ld, %ld\r\n", MAT33_ELMT(*m, 1, 0), MAT33_ELMT(*m, 1, 1), MAT33_ELMT(*m, 1, 2));
+  chprintf(sh, "    %ld, %ld, %ld\r\n", MAT33_ELMT(*m, 2, 0), MAT33_ELMT(*m, 2, 1), MAT33_ELMT(*m, 2, 2));
+}
+
+static void cmd_imu(shell_stream_t *sh, int argc, const char *const argv[])
+{
+  (void) argv;
+  int i = 0;
+  if (argc > 0) {
+    chprintf(sh, "Usage: rtos_mon\r\n");
```
Comment (fvantienen) [inline sw/airborne/modules/imu/imu.c:367]: This says `rtos_mon` and should be `imu`

Comment (gautierhattenberger) [reply sw/airborne/modules/imu/imu.c]: copy/paste curse

**Review by fvantienen — APPROVED** [review]

**Pull request merged**